modules/osdk-upgrading-v130-to-v180.adoc

```diff
@@ -0,0 +1,895 @@
+-  name: default
++  name: controller-manager
+   namespace: system
+----
+====
++
+.`config/rbac/role_binding.yaml` file diff
+[%collapsible]
+====
+[source,diff]
+----
+...
+   name: manager-role
+ subjects:
+ - kind: ServiceAccount
+-  name: default
++  name: controller-manager
+   namespace: system
+----
+====
++
+.`config/rbac/service_account.yaml` file diff
+[%collapsible]
+====
+[source,diff]
+----
++apiVersion: v1
++kind: ServiceAccount
++metadata:
++  name: controller-manager
++  namespace: system
+----
+====
+
+. Make the following changes to your `config/manifests/kustomization.yaml` file:
+
+.. Add a link:https://kustomize.io/[Kustomize] patch to remove the link:https://cert-manager.io/[cert-manager] `volume` and `volumeMount` objects from your cluster service version (CSV).
++
+Operator Lifecycle Manager (OLM) does not yet support cert-manager, so a JSON patch was added to remove this volume and mount so OLM can itself create and manage certificates for your Operator.
++
+In the `config/manifests/kustomization.yaml` file, add the following lines:
++
+.`config/manifests/kustomization.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+patchesJson6902:
+- target:
+    group: apps
+    version: v1
+    kind: Deployment
+    name: controller-manager
+    namespace: system
+  patch: |-
+    # Remove the manager container's "cert" volumeMount, since OLM will create and mount a set of certs.
+    # Update the indices in this path if adding or removing containers/volumeMounts in the manager's Deployment.
+    - op: remove
+      path: /spec/template/spec/containers/1/volumeMounts/0
+    # Remove the "cert" volume, since OLM will create and mount a set of certs.
+    # Update the indices in this path if adding or removing volumes in the manager's Deployment.
+    - op: remove
+      path: /spec/template/spec/volumes/0
+----
+====
+
+.. Optional: For Ansible- and Helm-based Operator projects, configure `ansible-operator` and `helm-operator` with a component config. To add this option, follow these steps:
+
+... Create the following file:
++
+.`config/default/manager_config_patch.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        args:
+        - "--config=controller_manager_config.yaml"
+        volumeMounts:
+        - name: manager-config
+          mountPath: /controller_manager_config.yaml
+          subPath: controller_manager_config.yaml
+      volumes:
+      - name: manager-config
+        configMap:
+          name: manager-config
+----
+====
+
+... Create the following file:
++
+.`config/manager/controller_manager_config.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+kind: ControllerManagerConfig
+health:
+  healthProbeBindAddress: :6789
+metrics:
+  bindAddress: 127.0.0.1:8080
+
```

Comment:
YAML with a blank line?  As long as you feel like customers can recognize what to specify for the `resourceName` field, then it's cool by me.  I don't know the subject area, so _I'm_ unsure.

Reply:
Hm yeah I'm not sure either. Came from https://github.com/operator-framework/operator-sdk/blob/2a1bd03/testdata/ansible/memcached-operator/config/manager/controller_manager_config.yaml.

I'll change the value to a user-replaceable `<resource_name>` for now. @rashmigottipati @varshab1210 Do you have a suggestion?